sample/src/App.tsx

```diff
@@ -26,8 +26,8 @@ Sentry.init({
   // Replace the example DSN below with your own DSN:
   dsn: SENTRY_INTERNAL_DSN,
   debug: true,
-  beforeSend: (e: Sentry.Event) => {
-    console.log('Event beforeSend:', e);
+  beforeSend: (e, hint) => {
+    console.log('Event beforeSend:', e, 'hint:', hint);
```

Comment:
L: you removed the `hint` in another PR, are you sure about this?

Reply:
Thanks. I definitely don't want this back.